packages/bsky/src/api/app/bsky/feed/getAuthorFeed.ts

```diff
@@ -99,9 +99,21 @@ export const skeleton = async (
           .whereRef('post_embed_image.postUri', '=', 'feed_item.postUri'),
       )
   } else if (filter === 'posts_no_replies') {
-    feedItemsQb = feedItemsQb.where((qb) =>
-      qb.where('post.replyParent', 'is', null).orWhere('type', '=', 'repost'),
-    )
+    feedItemsQb = feedItemsQb
+      .where((qb) =>
+        qb.where('post.replyParent', 'is', null).orWhere('type', '=', 'repost'),
+      )
+      .orWhere((qb) =>
```

Comment:
i think this `.orWhere` needs to make it's way into a `.where` wrapper. otherwise it might conflict with some other where statements (like limiting originator did to the author)

Reply:
actually the `where('originatorDid', '=', actorDid)` is outside of this whole if block

so you might just be able to tag that whereExists into the previous where clause as an `orWhereExists`

Reply:
Just to toss out another option, here avoiding the subquery/join:
```ts
qb.where('post.replyParent', 'is', null)
  .orWhere('post.replyRoot', 'like', `at://${actorDid}/%`)
  .orWhere('type', '=', 'repost'),
```

Reply:
Yep good call, can consolidate these. And good idea on the `like` check!